ls1intum/Artemis
# Show correct count for complaints/more feedback requests about your assessments

### Checklist

### Motivation and Context


We want to show all complaints/more feedback requests for current tutor (not only accepted).
Fixes #1084 

### Description

Previously number under the complaints about your assessments was showing number of accepted complaints for current tutor. Now it shows all the complaints (accepted/rejected/not resolved) for current tutor.
The same changes are applied to more feedback requests.
I also used primitive types whenever possible for TutorLeaderboard. This also allowed to simplify some if statements.
**UPDATE**: Changes to the database are not destructive anymore. Instead of killing the old views, I only create the new ones. The old ones can be killed later then

**UPDATE 2**: I found a way to test views and bring 100% Test coverage for `TutorLeaderboardService`.
![Screenshot_2020-03-16 de tum in www1 artemis service](https://user-images.githubusercontent.com/17262376/76805073-2fc40e80-67de-11ea-9633-a394dcc6d875.png)
### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Select Tutor Dashboard
4. Check that number of complaints/feedback requests about your assessments are shown correctly ( it should correspond to the number of complaints/requests shown on list of complaints/requests page when you click on the number)
5. Go to the exercise and check if Tutor Exercise Dashboard also has the correct number of complaints/ more feedback requests about your assessments.

### Screenshots


Correct number on Tutor Course Dashboard
![Screenshot_2019-12-14 Tutor Course Dashboard](https://user-images.githubusercontent.com/17262376/70852148-4d5fa000-1e9e-11ea-9dfe-b33881f66596.png)
List of complaints for this tutor
![Screenshot_2019-12-14 List of complaints](https://user-images.githubusercontent.com/17262376/70852153-5badbc00-1e9e-11ea-8524-16d97d2683e3.png)
